go.mod

```diff
@@ -22,20 +22,21 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
-	golang.org/x/sys v0.0.0-20210817190340-bfb29a6856f2
+	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
+	golang.org/x/sys v0.0.0-20210831042530-f4d43177bf5e
 	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.22.4
-	k8s.io/apimachinery v0.22.4
+	google.golang.org/protobuf v1.27.1
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/cloud-provider v0.22.4
-	k8s.io/klog/v2 v2.10.0
+	k8s.io/cloud-provider v0.23.0
+	k8s.io/klog/v2 v2.30.0
```

Comment:
I do not think you can update `klog` yet, there is an issue with `controller-runtime`?

Reply:
Yes, correct. reverted this one but facing other issues. reworking on this one will update this once everything works locally.

Reply:
controller runtime is updated to kube 1.23.0 in master. updated controller runtime to fix the klog issue.